media/css/firefox/whatsnew/whatsnew-142.scss

```diff
@@ -0,0 +1,866 @@
+  --fds-font-size-4xl: 2.25rem;   // 36px
+  --fds-font-size-5xl: 3rem;      // 48px
+
+  // Spacing (derived from Figma scales)
+  // Define missing Figma scale tokens used below
+  --figma-Scale-8: 8px;
+  --figma-Scale-16: 16px;
+  --figma-Scale-24: 24px;
+  --figma-Scale-32: 32px;
+
+  // Map Figma scale tokens to spacing variables
+  --fds-spacing-xs: calc(var(--figma-Scale-8) / 2); // 4px
+  --fds-spacing-sm: var(--figma-Scale-8);           // 8px
+  --fds-spacing-md: var(--figma-Scale-16);          // 16px
+  --fds-spacing-lg: var(--figma-Scale-24);          // 24px
+  --fds-spacing-xl: var(--figma-Scale-32);          // 32px
+  --fds-spacing-2xl: calc(var(--figma-Scale-32) * 1.5); // 48px
+  --fds-spacing-3xl: calc(var(--figma-Scale-32) * 2);   // 64px
+  --fds-spacing-4xl: calc(var(--figma-Scale-32) * 2.5); // 80px (section rhythm)
+
+  // Border radius
+  --fds-border-radius-sm: 4px;
+  --fds-border-radius-md: 8px;
+  --fds-border-radius-lg: 12px;
+  --fds-border-radius-xl: 16px;
+
+  // Shadows
+  --fds-shadow-sm: 0 1px 2px 0 rgba(0, 0, 0, 0.05);
+  --fds-shadow-md: 0 4px 6px -1px rgba(0, 0, 0, 0.1);
+  --fds-shadow-lg: 0 10px 15px -3px rgba(0, 0, 0, 0.1);
+  --fds-shadow-xl: 0 20px 25px -5px rgba(0, 0, 0, 0.1);
+
+  // Transitions
+  --fds-transition-fast: 0.15s ease;
+  --fds-transition-base: 0.2s ease;
+  --fds-transition-slow: 0.3s ease;
+}
+
+// Dark mode overrides
+@media (prefers-color-scheme: dark) {
+  :root {
+    --fds-color-white: var(--figma-Neutrals-Black);
+    --fds-color-gray-10: var(--figma-Neutrals-Charcoal);
+    --fds-color-gray-20: var(--figma-Neutrals-Grey-8);
+    --fds-color-gray-30: var(--figma-Neutrals-Grey-7);
+    --fds-color-gray-40: var(--figma-Neutrals-Grey-6);
+    --fds-color-gray-50: var(--figma-Neutrals-Grey-5);
+    --fds-color-gray-60: var(--figma-Neutrals-Grey-4);
+    --fds-color-gray-70: var(--figma-Neutrals-Grey-3);
+    --fds-color-gray-80: var(--figma-Neutrals-Grey-2);
+    --fds-color-gray-90: var(--figma-Neutrals-Gray-1);
+    --fds-color-black: var(--figma-Neutrals-White);
+    --fds-color-charcoal: var(--figma-Neutrals-Ash);
+    --fds-color-ash: var(--figma-Neutrals-Charcoal);
+  }
+}
+
+/* Dark/Light mode overrides */
+:root[data-theme="dark"] {
+  --fds-color-white: var(--figma-Neutrals-Black);
+  --fds-color-gray-10: var(--figma-Neutrals-Charcoal);
+  --fds-color-gray-20: var(--figma-Neutrals-Grey-8);
+  --fds-color-gray-30: var(--figma-Neutrals-Grey-7);
+  --fds-color-gray-40: var(--figma-Neutrals-Grey-6);
+  --fds-color-gray-50: var(--figma-Neutrals-Grey-5);
+  --fds-color-gray-60: var(--figma-Neutrals-Grey-4);
+  --fds-color-gray-70: var(--figma-Neutrals-Grey-3);
+  --fds-color-gray-80: var(--figma-Neutrals-Grey-2);
+  --fds-color-gray-90: var(--figma-Neutrals-Gray-1);
+  --fds-color-black: var(--figma-Neutrals-White);
+  --fds-color-charcoal: var(--figma-Neutrals-Ash);
+  --fds-color-ash: var(--figma-Neutrals-Charcoal);
+}
+
+:root[data-theme="light"] {
```

Comment:
Given the support profile for these pages, something like `light-dark()` can be easily used around here.